streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -387,6 +389,12 @@
     public static final String DEFAULT_PRODUCTION_EXCEPTION_HANDLER_CLASS_CONFIG = "default.production.exception.handler";
     private static final String DEFAULT_PRODUCTION_EXCEPTION_HANDLER_CLASS_DOC = "Exception handling class that implements the <code>org.apache.kafka.streams.errors.ProductionExceptionHandler</code> interface.";
 
+    /** {@code default.store.impl.class} */
+    @SuppressWarnings("WeakerAccess")
+    public static final String DEFAULT_STORE_IMPLEMENTATION_CLASS_CONFIG = "default.store.impl.class";
+    private static final String DEFAULT_STORE_IMPLEMENTATION_CLASS_DOC = "Store supplier implementation class to use. Default is set as RocksDBStoreImplementation. " +
+        "It can be overwritten dynamically during streaming operation.";
```

Comment:
(1) create a stream config: default.store.impl.class to store the default store implementation class, and default to RocksDB